libs/net/template/Models/Reports/TemplateModel.cs

```diff
@@ -42,7 +44,7 @@ public TemplateModel(IEnumerable<ContentModel> content)
     /// Creates a new instance of a TemplateModel, initializes with specified parameters.
     /// </summary>
     /// <param name="sections"></param>
-    public TemplateModel(Dictionary<string, ReportSectionModel> sections)
+    public TemplateModel(Dictionary<string, ReportSectionModel> sections, string? uploadPath = null)
```

Comment:
Remove invalid code

Reply:
This part of code is still valid for the moment, and we can chat more about it tomorrow morning if needed, thanks